### DISCUSSION TITLE
How to set up with tailscale, no video receiving so far

### DISCUSSION THREAD

**Discussion by Spyridion** [mrlt8/docker-wyze-bridge#1151] [Q&A]
Hi,

I'm pretty confused on how to set up the settings to go through tailscale's port number to access the cameras. I set up my firewall so only tailscale's port is open to the public. Any help would be appreciated. Thank you.

**Comment by Spyridion** [comment]
Is there perhaps a port number I need to whitelist for lan mode to work? Also, is it required to expose ports 443 to access the wyze apis? Can I set this app to go through tailscale's port?

**Comment by mrlt8** [comment]
I added a sample docker-compose for Tailscale: https://github.com/mrlt8/docker-wyze-bridge/blob/main/docker-compose.tailscale.yml
